etc/requirements.txt

```diff
@@ -4,3 +4,4 @@ pandas >= 0.15.0
 toolz >= 0.7.3
 multipledispatch >= 0.4.7
 networkx >= 1.0
+dask >= 0.11.1
```

Comment:
should this still be `dask[complete]` to get the reqs for dask dataframe and stuff?